src/slams/viny/TBM_prob_conversion.h

```diff
@@ -0,0 +1,39 @@
+#ifndef SLAM_CTOR_SLAM_VINY_TBM_PROB_CONVERSION_H
+#define SLAM_CTOR_SLAM_VINY_TBM_PROB_CONVERSION_H
+
+#include "../../core/maps/transferable_belief_model.h"
+#include "../../core/states/state_data.h"
+
+// TBM ---> Occupancy
+Occupancy TBM_to_O(const TBM& tbm) {
+  double qual = tbm.occupied() + tbm.empty();
```

Comment:
Can we put these viny cell model-related changes to a separate pull request/commit to ease maintenance?

Reply:
Good idea. Shall I open two new pull requesets? Or how to separate commits considering that the commits have already happened? I am not really familiar with git. Any instructions would be appreciated : )

Reply:
> Shall I open two new pull requests?

Yes, you need two pull requests: one with credibilist implementation, the other with viny changes.

> Or how to separate commits considering that the commits have already happened?

It can be done with `git rebase` (in your local commits history first) since (as I understand) you need just to rearrange commits. I'm not sure if such local history changes would be applied correctly to the pull request. If they are not just close this pull request and open the two new ones.
Maybe this example will be helpful http://blog.dennisrobinson.name/reorder-commits-with-git/.

And there is also a simpler option: revert viny model, make a commit, update pull request; commit the changes and create a new pull request.